tests/test_ingest.py

```diff
@@ -204,26 +215,27 @@ def test_ingest_local_dense_matrix(self):
         ]
         ingest = self.setup_ingest(args)
 
-        models = ingest.load_expression_data_args[0]
-
-        # Verify that 19 gene models were passed into load method
-        num_models = len(models)
-        expected_num_models = 19
-        self.assertEqual(num_models, expected_num_models)
+        genes = ingest.db.collection(u'genes')
+        stream = (
+            genes.where(u'taxon_common_name', u'==', u'human')
+            .where(u'study_accession', u'==', study_accession)
+            .stream()
+        )
 
-        # Verify that the first gene model looks as expected
-        mock_dir = "dense_matrix_19_genes_100k_cells_txt"
-        model, expected_model = get_nth_gene_models(0, models, mock_dir)
+        docs = [doc.to_dict() for doc in stream]
 
-        self.assertEqual(model, expected_model)
+        num_docs = len(docs)
+        self.assertEqual(num_docs, 19)
 
     def test_ingest_missing_local_file(self):
         """Ingest Pipeline should throw error for missing local file
         """
 
+        study_accession = get_random_study_accession()
+
         args = [
```

Comment:
You have double args here as well.

Reply:
This is the diff UI issue [described above](https://github.com/broadinstitute/scp-ingest-pipeline/pull/26#discussion_r325794668); there's actual just one set of args.